cli/src/pcluster/config/cluster_config.py

```diff
@@ -1260,6 +1264,19 @@ def is_dcv_enabled(self):
         """Return True if DCV is enabled."""
         return self.head_node.dcv and self.head_node.dcv.enabled
 
+    @property
+    def are_all_security_groups_customized(self):
+        """Return True if all of head node and queues have (additional) security groups specified."""
+        head_node_networking = self.head_node.networking
+        if not (head_node_networking.security_groups or head_node_networking.security_groups):
```

Comment:
this must be `additional_security_groups`.

Reply:
Fixed. Sorry